feat: add status bar managment (#642)
software-mansion/react-native-screens: native-stack/README.md

Defaults to `default`.

### Status bar managment

With `native-stack`, the status bar can be managed by `UIViewController` on iOS. It requires:

> 1. Enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file (and so disables the option to use React Native's `StatusBar` component). 

Author: ```suggestion
1. Enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file (it disables the option to use React Native's `StatusBar` component). 
```
